staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -122,8 +122,13 @@ func WithPriorityAndFairness(
 			served = true
 			innerCtx := context.WithValue(ctx, priorityAndFairnessKey, classification)
 			innerReq := r.Clone(innerCtx)
+
+			// We intentionally set the UID of the flow-schema and priority-level instead of name.
+			// This is so that no PII escapes through these headers, also the names might make it obvious to
```

Comment:
Strike the bit about PII and provide an example.  As I recall it, we originally chose to hide this just so that the names that cluster-admins chose for categorization were not exposed.

Reply:
@deads2k updated the comment.